pkg/3scale/amp/operator/system_mysql_options_provider.go

```diff
@@ -122,9 +112,9 @@ func (s *SystemMysqlOptionsProvider) systemDatabaseURLIsValid(rawURL string) (*u
 	if resultURL.User.Username() == "" {
 		return nil, fmt.Errorf("authentication information in '%s' field of '%s' secret must contain a username", component.SystemSecretSystemDatabaseURLFieldName, component.SystemSecretSystemDatabaseSecretName)
 	}
-	if resultURL.User.Username() != "root" {
-		return nil, fmt.Errorf("authentication information in '%s' field of '%s' secret must contain 'root' as the username", component.SystemSecretSystemDatabaseURLFieldName, component.SystemSecretSystemDatabaseSecretName)
-	}
+	//if resultURL.User.Username() != "root" {
+	//	return nil, fmt.Errorf("authentication information in '%s' field of '%s' secret must contain 'root' as the username", component.SystemSecretSystemDatabaseURLFieldName, component.SystemSecretSystemDatabaseSecretName)
+	//}
```

Comment:
```suggestion
```
Should this commented out section be removed?

Reply:
updated/removed. Thank you